src/components/typography/link/size.json

```diff
@@ -0,0 +1,10 @@
+{
+  "size": {
+    "link": {
+      "focus": {
+        "border-width": { "value": "{size.border.width.thin}" },
+        "border-radius": { "value": "0.125rem" }
```

Comment:
This `border-radius` size doesn't have a global token, because it's used just here. Should we create one and give it a name or leave it this way?